prep-javascript-functions/main.js

```diff
@@ -0,0 +1,62 @@
+function addTwoNumbers(num1, num2) {
+  var result = (num1 + num2);
+  return result;
+}
+var sumTwoNumbers = addTwoNumbers(2, 2);
+console.log('sumTwoNumbers:', sumTwoNumbers);
+
+function convertHoursToMinutes(hours) {
+  var result = (hours) * 60;
+  return result;
+}
+var HoursToMinutes = convertHoursToMinutes(2);
+console.log('HourstoMinutes:', HoursToMinutes);
+
+function getGreeting(name) {
+  var result = ('hello world');
+  return result;
+}
+var greeting = getGreeting('Hello World');
+console.log('greetings:', greeting);
```

Comment:
We can concatenate the string `'Hello '` before the `name` parameter to get a greeting for any name.

```js
function getGreeting(name) {
  var result = 'Hello ' + name + '!';
  return result;
}

var greeting = getGreeting('Gabrielle');
console.log('greetings:', greeting);
```